packages/conventional-gitmoji/src/types/types.types.ts

```diff
@@ -0,0 +1,97 @@
+// @lol at this filename
+
+type CommitTypes =
```

Comment:
Yes, funny filename, haha. Please give some more thought to this.

Also, let's see about time checking how long it would take to auto-generate this with a script. At one point this was auto-generated and we imported a JSON file (or `js` I can't remember).

Then it was moved to just a dynamic object creation which made it kind of vague as to what all this was doing.

It makes the package bigger, however, it is much easier to debug if these are present.